### ISSUE TITLE
The adult microbiome of healthy and otitis patients: Definition of the core healthy and diseased ear microbiomes

### ISSUE THREAD

**Issue by SvetlanaUP** [waldronlab/BugSigDBcuration#189]
The adult microbiome of healthy and otitis patients: Definition of the core healthy and diseased ear microbiomes – Maria Burton et al. – PlosOne
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0262806

**Comment by alcantara-m** [comment]
Hello! Could I be assigned to this project? Thank you @SvetlanaUP :)

**Comment by FaithAlexander** [comment]
Hello @SvetlanaUP,  I would love to work on this paper. Can I have it assigned to me please?

**Comment by Msjulieozi** [comment]
Can I please be assigned to this project? @SvetlanaUP , I would love to work on it. Thanks you!

**Comment by SvetlanaUP** [comment]
Assigned to @alcantara-m

**Comment by AleruDivine** [comment]
Hey @alcantara-m, good morning! I hope you're doing well.
I noticed this issue has been assigned to you for a while now, and since we're in the final week of contributions, I just wanted to check in. If you need a hand with anything, if you have any questions or blockers, just give me a shout. I'll be more than happy to help out however I can! 😊

**Comment by samreen-19** [comment]
Hi @SvetlanaUP I noticed that this paper hasn't even been opened on bugsigdb yet, and I've finished my earlier curation. So may I work on it?

**Comment by AleruDivine** [comment]
@samreen-19, this paper is currently being curated by @alcantara-m

**Comment by samreen-19** [comment]
Oh okay, thank you for informing @AleruDivine

**Comment by samreen-19** [comment]
@AleruDivine are you sure tho? I looked the study up on bugsigdb , and I didn't find any existing study for this paper. 😅

**Comment by AleruDivine** [comment]
Here is the link: https://bugsigdb.org/Study_1069

**Comment by samreen-19** [comment]
Ohh, my bad. I apologize.

**Comment by AleruDivine** [comment]
That's okay @samreen-19

**Comment by BarakatAA** [comment]
Hi @alcantara-m good morning and trust you're doing great. 

I noticed this study does not yet have the 'needs review' tag and wanted to know if you needed help. Do reach out if so. 
All best wishes!

**Comment by AleruDivine** [comment]
@alcantara-m I understand you, you also need to let the mentors know by commenting here that your paper is ready for review, so they can add the "needs review" tag here too

**Comment by alcantara-m** [comment]
Hi @SvetlanaUP the paper is ready since April 2nd!

Thank you @AleruDivine for the heads up!

**Comment by BarakatAA** [comment]
Hi @alcantara-m  I meant here on Github. It lacks the tag so I thought maybe you were still curating. If you're done curating then mention it here and tag the mentors

**Comment by BarakatAA** [comment]
Okay, I see you've that now.

**Comment by Scholarpat** [comment]
Hi @SvetlanaUP,

This issue has been resolved. All the experiments and signatures are now displaying. 
I've reviewed the curation again and here are the changes that was made here;

- The body site, group names, antibiotic exclusion and sequencing type were not properly curated for experiment 1 and 2. I've corrected these and included their signatures.
- Curated additional experiments (3 - 6) and their signatures as well.

This curation is now complete. Thank you!
